src/screens/AppAppearence/AppAppearence.js

```diff
@@ -0,0 +1,147 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import t from 'translations/translate';
+import styled, { useTheme } from 'styled-components/native';
+
+// Components
+import { Container, Center } from 'components/layout/Layout';
+import HeaderBlock from 'components/HeaderBlock';
+import Text from 'components/core/Text';
+import Button from 'components/core/Button';
+import Image from 'components/Image';
+
+// Constants
+import { DARK_THEME, LIGHT_THEME } from 'constants/appSettingsConstants';
+
+// Utils
+import { fontStyles, spacing, appFont } from 'utils/variables';
+import { getThemeColors } from 'utils/themes';
+
+// Actions
+import { setAppThemeAction } from 'actions/appSettingsActions';
+
+// Assets
+const lightTheme = require('assets/images/appAppearence/lightTheme.png');
+const darkTheme = require('assets/images/appAppearence/darkTheme.png');
+
+const AppAppearence = () => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const theme = useTheme();
+  const { current } = theme;
+  const colors = getThemeColors(theme);
+  const [isLightThemePressed, setLightThemePressed] = useState(current === LIGHT_THEME);
+  const [isDarkThemePressed, setDarkThemePressed] = useState(current === DARK_THEME);
+
+  const onPressLightTheme = () => {
+    setLightThemePressed(true);
+    setDarkThemePressed(false);
+    dispatch(setAppThemeAction(LIGHT_THEME, true));
+  };
+
+  const onPressDarkTheme = () => {
+    setDarkThemePressed(true);
+    setLightThemePressed(false);
+    dispatch(setAppThemeAction(DARK_THEME, true));
+  };
+
+  const selectedThemeStyle = {
+    borderColor: colors.buttonPrimaryBackground,
+    borderWidth: 2,
+    borderRadius: 10,
+  };
```

Comment:
Please remember to move this down to the other style definitions (around line 106)